Wikipedia/Code/WMFImageTagParser.h

```diff
@@ -8,6 +8,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (WMFImageTagList*)imageTagListFromParsingHTMLString:(NSString*)HTMLString;
 
+- (WMFImageTagList*)imageTagListFromParsingHTMLString:(NSString*)HTMLString withLeadImageURL:(nullable NSURL *)leadImageURL;
```

Comment:
@joewalsh I'm not sure we need "withLeadImageURL:" - the reason is the parser reduces the html you pass it to just image tags, which means if you just add an img tag for the lead image to the beginning of the HTMLString it should just work.